csunplugged/static/scss/sticky-state.scss

```diff
@@ -0,0 +1,26 @@
+/*!
+ * sticky-state 2.4.1 - StickyState is a high performant module making native position:sticky statefull and polyfill the missing sticky browser feature
+ * Copyright (c) 2016 Sönke Kluth <soenke.kluth@gmail.com> (http://soenkekluth.com/) - https://github.com/soenkekluth/sticky-state#readme
+ * License: MIT
+ */
+.sticky {
+  position: -webkit-sticky;
+  position: sticky;
+}
+.sticky.sticky-fixed.is-sticky {
+  margin-top: 0;
+  margin-bottom: 0;
+  position: fixed;
+  -webkit-backface-visibility: hidden;
+  -moz-backface-visibility: hidden;
+  backface-visibility: hidden;
```

Comment:
Yep, looks like it's currently only supported with prefix (https://developer.mozilla.org/en-US/docs/Web/CSS/backface-visibility#Browser_compatibility) so this is a future proofed CSS rule. Should be fine to leave as is.